lib/safe-pg-migrations/helpers/statements_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module SafePgMigrations
+  module Helpers
+    module StatementsHelper
+      RETRIABLE_SCHEMA_STATEMENTS = %i[
```

Comment:
Name is **confusing**. It's not a list of retriable statements but statements on which we produce logs. 
Do we even need to have an include list ? Why not enable this for every migrations ?

Reply:
good question indeed, I'm not sure
Maybe some statement would be too costly I'm not sure

I think this is more a "those ones are more eager to fail" approach 

Not sure how to retrieve migrations statements; probably with `ActiveRecord::ConnectionAdapters::SchemaStatements.instance_methods(false)`

But then not sure all of them should be logged or retried